test/cli/stdin_and_name.stdin

```diff
@@ -0,0 +1 @@
+sample/a.ml
```

Comment:
can you add "Closes #1112" to this commit?

Reply:
That's not really testing editor integration but I agree that's enough.